config/past_chancellors/content_item.yml

```diff
@@ -1,9 +1,12 @@
 ---
-base_path: /government/history/past-foreign-secretaries
+base_path: /government/history/past-chancellors
 title: Past Chancellors of the Exchequer
```

Comment:
Why is this changing?

Reply:
Looks from the file name that the original base-path was a copy/pasta error

Reply:
Moved to #3690.